api.raml

```diff
@@ -33,6 +33,12 @@ baseUri: http://api.circular.works
                             "id": "25"
                           }
                         ]
+                      },
+                      "author": {
+                        "data": {
+                          "type": "users",
+                          "id": "3"
+                        }
```

Comment:
Probably not. We have to check how to serialize different data in different points though.